hourglass_blocks.py

```diff
@@ -0,0 +1,193 @@
+import keras.backend as K
+from keras.layers import *
+from keras.losses import mean_squared_error
+from keras.models import *
+from keras.optimizers import Adam, RMSprop
+
+# Adapted from https://github.com/yuanyuanli85/Stacked_Hourglass_Network_Keras/blob/master/src/net/hg_blocks.py
+
+def create_hourglass_network(num_classes, num_stacks, num_channels, inres, outres, bottleneck):
+    input = Input(shape=(inres[0], inres[1], 3))
```

Comment:
Configurable input size, 3 channels, any idea what dimensions would work best for this arch?

Reply:
yep, these are given in `train.py`. Num classes is the number of joints, so for us it is 17. Stacks should be 2-8, with the paper we are basing it off of using 8.